backend/src/main/java/com/huellapositiva/application/controller/ESALApiController.java

```diff
@@ -99,6 +104,52 @@ public void registerESAL(@Validated @RequestBody ESALRequestDto dto,
         }
     }
 
+    @Operation(
+            summary = "Updates the information of an ESAL",
+            description = "Updates the information of the ESAL linked to the logged employee. Roles allowed CONTACT_PERSON and CONTACT_PERSON_NOT_CONFIRMED.",
+            tags = "ESAL",
+            parameters = {
+                    @Parameter(name = "X-XSRF-TOKEN", in = ParameterIn.HEADER, required = true, example = "a6f5086d-af6b-464f-988b-7a604e46062b", description = "For take this value, open your inspector code on your browser, and take the value of the cookie with the name 'XSRF-TOKEN'. Example: a6f5086d-af6b-464f-988b-7a604e46062b"),
+                    @Parameter(name = "XSRF-TOKEN", in = ParameterIn.COOKIE,required = true, example = "a6f5086d-af6b-464f-988b-7a604e46062b", description = "Same value of X-XSRF-TOKEN")
+            },
+            security = {
+                    @SecurityRequirement(name = "accessToken")
+            }
+    )
+    @ApiResponses(
+            value = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            description = "Ok, ESAL updated successfully."
+                    ),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "Bad Request, unable to validate provided data."
+                    ),
+                    @ApiResponse(
+                            responseCode = "409",
+                            description = "Conflict, the provided name is already taken."
+                    ),
+                    @ApiResponse(
+                            responseCode = "500",
+                            description = "Internal server error, could not update the esal.",
+                            content = @Content(mediaType = "application/json")
+                    )
+            }
+    )
+    @PostMapping("/update")
```

Comment:
@_PutMapping("/") -> as a general rule we use POST for creation and PUT for updates

Reply:
Solved